calico/felix/config.py

```diff
@@ -463,7 +469,51 @@ def _validate_cfg(self, final=True):
         if self.ETCD_CA_FILE == "none":
             self.ETCD_CA_FILE = None
 
-        if self.ETCD_SCHEME == "https":
+        # Determine which etcd scheme and addresses will be used.
+        addrs = []
```

Comment:
I'm not really a fan: it does have some interaction with the rest of the function; and I think the break in attention (from having to move to a different place in the file) has more readability impact than the extra length of having it inline.